src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Cohort.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+using SFA.DAS.Authorization;
+
+namespace SFA.DAS.ProviderCommitments.Web.Models
+{
+    public class Cohort : IAuthorizationContextModel
```

Comment:
This should probably be suffixed with `Request` and moved to the `SFA.DAS.ProviderCommitments.Web.Requests` namespace to follow the project conventions. It also looks like we currently have three types that are inputs to the various `AddDraftApprenticeship` action methods:

1. `Cohort`
2. `AddDraftApprenticeship`
3. `ReservationsAddDraftApprenticeship`

What do you think to renaming these to something along the lines of:

1. `NonReservationsAddDraftApprenticeship`
2. `CreateCohortReuqest`
3. `ReservationsAddDraftApprenticeship`

(1 & 3 may also be able to be consolidated, see comment above).

Reply:
I agree with the naming. But I think 2 should be renamed to 'CreateCohortWithDraftApprenticeshipRequest' as there

Reply:
Also I've renamed the controller to a matching name 'CreateCohortWithDraftApprenticeshipController'